app/Http/Controllers/StatisticController.php

```diff
@@ -26,4 +27,67 @@ public function daily()
 
         return view('statistics.daily')->with($data);
     }
+
+    /**
+     * Quarterly statistics counting
+     *
+     * @param Illuminate\Http\Request $request request
+     *
+     * @return Illuminate\Http\Response
+     */
+    public function quarterly(Request $request)
+    {
+        if (!isset($request->quarter)) {
+            $quarter = date('Y') . 'Q' . (StatisticController::toQuarter(date('n')) - 1);
```

Comment:
@at-dungnh 
Because the current Quarter is not enough data to show statistics. So by default, we use the data of the previous Quarter to view.

For example: The current Quarter is III, the system will return data presentation of Quarter II.